### PR TITLE
[NETBEANS-3659] - clean up List raw type warnings..

### PR DESCRIPTION
**Pull request by BradWalker**
There are a lot of places where we get raw type warnings..

   [repeat] /home/bwalker/src/netbeans/platform/core.multiview/src/org/netbeans/core/multiview/MultiViewPeer.java:1070: warning: [rawtypes] found raw type: List
   [repeat]         private List listeners = new ArrayList();
   [repeat]                 ^
   [repeat]   missing type arguments for generic class List<E>
   [repeat]   where E is a type-variable:
   [repeat]     E extends Object declared in interface List

Clean up a bunch of these warnings..

### CODE REVIEW

File: java/beans/src/org/netbeans/modules/beans/PropertyPatternPanel.java [hunk 1]
```diff
@@ -519,7 +519,7 @@ public void actionPerformed( java.awt.event.ActionEvent e ) {
         if (TYPES == null) {
             String typeList = NbBundle.getMessage(PropertyPatternPanel.class, "IdxPropertyPatternPanel_Types"); // NOI18N
             StringTokenizer st = new StringTokenizer(typeList, "|"); // NOI18N
-            List l = new LinkedList();
+            List<String> l = new LinkedList();
```
Comment (matthiasblaesing) [inline java/beans/src/org/netbeans/modules/beans/PropertyPatternPanel.java]: The instanziation is missing a diamond operator.

File: ide/xml.multiview/src/org/netbeans/modules/xml/multiview/ui/SectionInnerPanel.java [hunk 2]
```diff
@@ -49,7 +49,7 @@
     private static final RequestProcessor RP = new RequestProcessor(SectionInnerPanel.class);
 
     private SectionView sectionView;
-    private java.util.List refreshableList = new LinkedList();
+    private java.util.List<Refreshable> refreshableList = new LinkedList();
```
Comment (matthiasblaesing) [inline ide/xml.multiview/src/org/netbeans/modules/xml/multiview/ui/SectionInnerPanel.java]: Missing Diamond.

File: java/beans/src/org/netbeans/modules/beans/PropertyPatternPanel.java (hunk 1 above)
Comment (BradWalker) [reply java/beans/src/org/netbeans/modules/beans/PropertyPatternPanel.java]: good eyes.. fixed..

File: ide/xml.multiview/src/org/netbeans/modules/xml/multiview/ui/SectionInnerPanel.java (hunk 2 above)
Comment (BradWalker) [reply ide/xml.multiview/src/org/netbeans/modules/xml/multiview/ui/SectionInnerPanel.java]: ditto.. fixed..

File: java/beans/src/org/netbeans/modules/beans/PropertyPatternPanel.java (hunk 1 above)
Comment (lalo-mx) [reply java/beans/src/org/netbeans/modules/beans/PropertyPatternPanel.java]: The list still accepts any objects. Is that wanted?

Comment (BradWalker) [reply java/beans/src/org/netbeans/modules/beans/PropertyPatternPanel.java]: Ugg.. Mismerge on my part.. Give me a second to fix.. Sorry about that..

Comment (BradWalker) [reply java/beans/src/org/netbeans/modules/beans/PropertyPatternPanel.java]: Should be fixed now..

**Comment by matthiasblaesing** [comment]
Thank you.

**Pull request merged**